dags/prio/kubernetes.py

```diff
@@ -0,0 +1,109 @@
+from datetime import timedelta
+from os import environ
+
+from airflow import DAG
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from operators.gcp_container_operator import (
+    GKEClusterCreateOperator,
+    GKEClusterDeleteOperator,
+    GKEPodOperator,
+)
+from utils.gke import create_gke_config
+from airflow.operators.bash_operator import BashOperator
+
+
+def container_subdag(
+    parent_dag_name,
+    child_dag_name,
+    default_args,
+    gcp_conn_id,
+    service_account,
+    server_id,
+    env_vars,
```

Comment:
should 'env_vars' and 'arguments' have empty dict/lists (respectively) as defaults so we don't need to pass in env vars = {} like line 114 of dags/prio/processor.py ?